scripts/cliupdate.php

```diff
@@ -57,6 +57,7 @@
 include_once (GLPI_ROOT . "/inc/db.function.php");
 include_once (GLPI_CONFIG_DIR . "/config_db.php");
 
+global $PHPLOGGER, $SQLLOGGER; //globals...
```

Comment:
I do not see why it is declared here.

Reply:
They must exists before they can be used/assigned; declaring a global at the beginning rather than in any further line does not sounds illogical to me.